test/utils.js

```diff
@@ -71,8 +71,24 @@ const fixtures = {
 				newData.content, newData.published, newData.type]
 		)
 		return newPost.rows[0]
+	},
+	async insertPostLike(overrides = {}){
+		const likeData = {
+			type : 'gif'
```

Comment:
this should not be a `gif` straightaway, 

it should give the possible values and be random. So  let's use `faker.helpers.arrayElement(['gif', 'post'])`. It will randomise between `gif` and `post`. Then the overrides will determine if it should be `gif` or `post`

Noticed the `insertPost` also just uses `gif`, kindly update that as well though

Reply:
yeah, that makes sense